setup/installer.py

```diff
@@ -14,10 +14,43 @@
 
 """ Forseti Installer.
 
-A stub to call gcp/install_setup.py which installs into GCP.
+A stub to call gcp/run_forseti_installer.py which installs into GCP.
 """
 
-from gcp import install_setup
+import site
+
+import pip
+
+
+INSTALLER_REQUIRED_PACKAGES = [
+    'ruamel.yaml'
+]
+
+def install(package_name):
+    """Install package.
+
+    Args:
+        package_name (str): Name of the package to install.
+    """
+    pip.main(['install', package_name, '--user'])
+
+
+def install_required_packages():
+    """Install required packages."""
+    installed_pkgs = [pkg.key for pkg in pip.get_installed_distributions()]
+    for package in INSTALLER_REQUIRED_PACKAGES:
+        if package not in installed_pkgs:
+            install(package)
+
+
 
 if __name__ == '__main__':
-    install_setup.run()
+    # We need to install all the required packages before importing our modules
+
+    # Installing required packages
```

Comment:
If this is a stub, why should these be here?  The only job of this stub is to alias off `run_forseti_installer.run()`

Reply:
I think the runner itself should not care about package dependencies. It's like running setup.py, it's usually done before running the application logic.